templates/.github/workflows/go-test.yml

```diff
@@ -37,8 +39,11 @@ jobs:
         uses: protocol/multiple-go-modules@v1.0
         with:
           run: go test -v -race ./...
+      - name: Collect coverage files
+        shell: bash
+        run: echo "COVERAGES=$(find . -type f -name 'module-coverage.txt' | tr -s '\n' ',' | sed 's/,$//')" >> $GITHUB_ENV
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@a1ed4b322b4b38cb846afb5a0ebfa17086917d27 # v1.5.0
         with:
-          file: coverage.txt
+          files: ${{ env.COVERAGES }}
```

Comment:
We might want to quote paths considering usage here. I am wondering what would happen if a path to coverage file contains space.

Reply:
OK looks like the action [simply splits](https://github.com/codecov/codecov-action/blob/3375d62f3d177b734d1845da60213c1466b4f777/src/buildExec.ts#L96-L100) by `,`. In this case we might want to quote this entire value to assert the expanded value is passed as a string?

Reply:
this is yaml though, not shell - do you think spaces matter?

Reply:
Yep it's yaml. I am trying to think how it could break; directory names with `:` and other yaml special characters for example? The fundamental point here is, we mean to set the value of `files` key to string literal. So why not express it through syntax.

Reply:
Yeah I agree, a quoted string is probably a bit safer.

Reply:
Spaces should be fine, and I do hope that we don't use any weird special characters in directory names. But quoting doesn't hurt, so let's do it now before we run into any problems.